internal/server/worker_auth.go

```diff
@@ -57,7 +83,7 @@ func newWorkerAuth(ctx context.Context, workerKeyIdentifier, workerId string, op
 		l.WorkerEncryptionPubKey = opts.withWorkerKeys.workerEncryptionPubKey
 	}
 	if &opts.withControllerEncryptionPrivateKey != nil {
-		l.ControllerEncryptionPrivKey = opts.withControllerEncryptionPrivateKey
+		l.CtControllerEncryptionPrivKey = opts.withControllerEncryptionPrivateKey
```

Comment:
Should this be the ciphertext field?

Reply:
Should we move it to one of the test files?

Reply:
not sure that's necessary, it is a `new` function after all, just that this is how its used. maybe worth while adding a comment that it assigns to the ciphertext field

Reply:
the more I thought about this, the less I liked leaving this as-is. leaving as is makes it so that (if it's ever used outside of tests) the default encrypt/decrypt functions are required to be used to create a new worker auth. I've updated new worker auth to assign to plaintext and removed the `withKeyId` option as well as updated the tests and test helpers that use this func to encrypt the worker auth afterwards to correctly assign the keyid and ciphertext fields